regtest/README.md

```diff
@@ -12,35 +12,4 @@ Based on [Boltz Regtest](https://github.com/BoltzExchange/regtest)
 ./stop.sh
 ```
 
-```bash
-./restart.sh
-```
-
-- Web App: [http://localhost:8080](http://localhost:8080)
-
-Data dirs for the services are stored in `./data` folder.
-
-### Scripts container
-
-```bash
-docker exec -it boltz-scripts bash
-```
-
-- bitcoin-cli-sim-client
-- bitcoin-cli-sim-server
-- elements-cli-sim-client
-- elements-cli-sim-server
-- boltzcli-sim ([boltz-client](https://github.com/BoltzExchange/boltz-client))
-- lightning-cli-sim
-- lncli-sim
-
-Since there are two lnd and two cln instances, use `lncli-sim 1` or `lightning-cli-sim 1` to interact with the first instance and `lncli-sim 2` or `lightning-cli-sim 2` to interact with the second.
-
-Or alternatively, you can `source aliases.sh` to have these convenience scripts available on the host machine.
-
-### Block explorers
-
-[Esplora](https://github.com/Blockstream/esplora) is running for the Bitcoin Core and Elements regtest:
-
-- Bitcoin: [http://localhost:4002](http://localhost:4002)
-- Elements: [http://localhost:4003](http://localhost:4003)
+See `boltz/README.md` for more info.
```

Comment:
Linky link?

Reply:
Added in [b637db6](https://github.com/breez/breez-sdk-liquid/pull/774/commits/b637db6c880338476825da3a2cca4e378a6a47a4)